### PR TITLE
Update as per meeting minutes

### PR DESCRIPTION
**Pull request by simontWork**
- Fixed dangling sentence on use of sub-black and super-white in video formats
- Changed wording on Tone-mapping to indicate it was permissive (original sentence was neither permissive for obligatory)
- Added note on full-range BT.709 images
- Added note on using mDCV to flag non-standard viewing conditions
- Changed sentence on tone-mapping for a target display - the mDCV and cLLI don't target specific displays, they may (or may not) augment tone-mapping for a range of displays 
- Removed sentence on this method of tone-mapping being subjecive.  They are all subjective with optimisations for hue preservatin, highlight detail, saturation etc.
- Added note on display metadata being most important for absolute luminance formats.

### CODE REVIEW

**Review by svgeesus — APPROVED** [review]
Overall some nice improvements. I have a question about the edition of BT.2390, but that need not hold up approval and merging.

File: index.html
Comment (chrisn) [inline index.html:3819]:
```suggestion
            The <a>HLG</a> [[ITU-R-BT.2100]] image format does have published methods for translating the image for both changes in display luminance (within [[ITU-R-BT.2100]]) and 
```

File: index.html
Comment (chrisn) [inline index.html:3820]:
```suggestion
              ambient illumination (within the accompanying report [[ITU-R-BT.2390]]). This may be used with SDR images.
```

**Review by chrisn — COMMENTED** [review]
I suggested a couple of minor tweaks, but overall looks good to me.

File: index.html
Comment (simontWork) [reply index.html:181]: There was no reason - happy to link to the latest version.

**Pull request merged**
